client-v2/src/shared/actions/Groups.js

```diff
@@ -0,0 +1,41 @@
+// @flow
```

Comment:
I understand why this has been renamed but I also want to use these for the clipboard which don't have groups. Maybe we could talk about `ContainerArticleFragments` to distinguish them from supporting article fragments?

Reply:
This sounds like a good plan - let's talk IRL 👍